jenkins/scripts/files/run_integration_tests.sh

```diff
@@ -45,8 +46,10 @@ fi
 
 if [ "${REPO_NAME}" == "metal3-dev-env" ]
 then
-   METAL3REPO="${UPDATED_REPO}"
-   METAL3BRANCH="${UPDATED_BRANCH}"
+   export METAL3REPO="${UPDATED_REPO}"
+   export METAL3BRANCH="${UPDATED_BRANCH}"
+   export BAREMETAL_OPERATOR_LOCAL_IMAGE="${M3PATH}/baremetal-operator"
+   export CAPM3_LOCAL_IMAGE="${M3PATH}/cluster-api-provider-metal3"
```

Comment:
If you give it this way, they have to exist beforehand. The other way is to give the url to the github repo

Reply:
Fixed